From 694f7c31e327825574dfee102d50b88494792dcd Mon Sep 17 00:00:00 2001
From: Vinitha V Pillai <vinitha.pillai@nxp.com>
Date: Mon, 8 Jan 2018 11:29:08 +0530
Subject: [PATCH 063/129] armv8: ls1088a: Compile off DCACHE for SPL size reduction

Enable CONFIG_SYS_DCACHE_OFF in case of SPL build as DCACHE is
not being enabled in case of SPL image but was compiled in to
add redundant code.

Signed-off-by: Sumit Garg <sumit.garg@nxp.com>
Signed-off-by: Vinitha V Pillai <vinitha.pillai@nxp.com>
---
 include/configs/ls1088a_common.h |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/include/configs/ls1088a_common.h b/include/configs/ls1088a_common.h
index 0ce65ae..ef66119 100644
--- a/include/configs/ls1088a_common.h
+++ b/include/configs/ls1088a_common.h
@@ -18,6 +18,7 @@
 #define SPL_NO_SATA
 #define SPL_NO_QSPI
 #define SPL_NO_IFC
+#define CONFIG_SYS_DCACHE_OFF
 #undef CONFIG_DISPLAY_CPUINFO
 #endif
 
-- 
1.7.1

